src/main/java/teammates/ui/controller/InstructorCourseAddAction.java

```diff
@@ -73,8 +71,8 @@ public ActionResult execute() {
         String courseNameToShowParam = "";
 
         if (isError) { // there is error in adding the course
-            courseIdToShowParam = SanitizationHelper.sanitizeForHtml(newCourse.getId());
-            courseNameToShowParam = SanitizationHelper.sanitizeForHtml(newCourse.getName());
+            courseIdToShowParam = SanitizationHelper.sanitizeForHtml(newCourseId);
+            courseNameToShowParam = SanitizationHelper.sanitizeForHtml(newCourseName);
 
             List<String> statusMessageTexts = new ArrayList<>();
```

Comment:
It's not touched by your PR, but can you refactor this `List` and `for` part to use a stream since we're moving to java 8 anyway?

Reply:
I think it should be left to other PRs because this appears in many Action classes and it should be standardized, i.e. Either leave them the way they are or change all of them at once.